.github/workflows/propagate_to_manifestdb.yml

```diff
@@ -0,0 +1,77 @@
+name: Propagate update to OSBuild
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout OSBuild
+        uses: actions/checkout@v3
+        with:
+          path: osbuild
+          fetch-depth: 0
+
+      - name: Update Manifest-db
+        run: |
+          # update only if manifest-db SHA was updated in the last commit
```

Comment:
This is slightly weird. In a rare-case, the `manifest-db` ref might get updated in a different commit than the last one (when a manual fix is needed). Is this something we want to care about? Not sure tbh.

Reply:
Yeah, this condition is to be sure that we are in a space in time where a successful update `manifest-db` that was built with `osbuild-vx1` is still valid with`osbuild-vlast`. Updating the `osbuild` hash in `manifest-db` just after updating the ref on `osbuild` is an easy way to synchronize both of them.